packages/react-native-payments/lib/ios/ReactNativePayments.m

```diff
@@ -90,6 +90,13 @@ - (NSDictionary *)constantsToExport
     callback(@[[NSNull null]]);
 }
 
+RCT_EXPORT_METHOD(openPaymentSetup: (RCTResponseSenderBlock)callback) {
+    PKPassLibrary* lib = [[PKPassLibrary alloc] init];
+    [lib openPaymentSetup];
```

Comment:
@chukohsin 

Sounds like you also need to include `DOMString merchantIdentifier` when calling this:

https://developer.apple.com/documentation/apple_pay_on_the_web/applepaysession/2710269-openpaymentsetup

Reply:
cc @joepasq @MiraEs @deemaabdallah

Reply:
@CuriousMark I think that's for Apple Pay for Web. The [mobile one](https://developer.apple.com/documentation/passkit/pkpasslibrary/1617100-openpaymentsetup) doesn't seem to have the parameter.

Reply:
@chukohsin is correct. @CuriousMark what you shared is the web implementation